src/Base/Base.cpp

```diff
@@ -37,40 +52,49 @@ void Base::CreateReadme(){
 		while(std::getline(ReadMeTemplate, CurrentLine)){
 			std::cout<<CurrentLine<<std::endl;
 			std::string ParsedString = ParseAndReplace(CurrentLine);
+            std::cout<<ParsedString<<std::endl;
 		}
 	} else {
 		std::cout<<" No file "<<std::endl;
 	}
-
-
-
-
 }
 
 std::string Base::ParseAndReplace(std::string RawLine){
-	int StrLen = RawLine.length();
-	std::string StartString, EndString; 
+	int i,j, StrLen = RawLine.length();
+	std::cout<<"Strlen = "<<StrLen<<std::endl;
+    bool Found = false;
+    std::string StartString, EndString, ParsedString = RawLine, TempKey; 
 	if (StrLen>4){
-		for (int i=0; i<StrLen-4; i++){
-				if (StartString == "{{") break;			
-		}
-		
+		for (i=0; i<StrLen-4; i++){
+            StartString = RawLine.substr(i,2);    

```

Comment:
Configure editor to have four space tab. You may use astyle formatter in the beginning for consistency and then use proper editor.